docs/CONTRIBUTING.md

```diff
@@ -54,3 +54,66 @@ Join the [OpenCodeAcademy.org discord server](https://discord.gg/feCpvsXFBf) (if
 7. Getting an issue assigned to you:
     - If you think you would like to get an issue assigned to you, comment that you would like to work on it, and we will assign it to you.
 8. PLEASE make sure you update/add documentation where appropriate
+
+## Structure of src folder
```

Comment:
Hey, so for this one, instead of saying where everything is, and how everything is organized, explicitly, try to write heuristics  (like bullets) that will help people with naming conventions and making decisions about project structure. 
For example, we have currently 
root (configurations, .git, project setup stuff, src directory) > src (contains the source code for the library) > components, core, etc. {high level classifications, think modules or packages of logically connected packages} >  source files (Colors.constants.ts, Button.component.ts)

Then, talk about the different file extensions, and why we are using "Button.component.js" and the semantic value that we get from that pattern.  

Does that make sense?

Reply:
Information regarding file naming conventions has already been included in the Guidelines section. Do you think it's still necessary to mention it again?
![Capture](https://user-images.githubusercontent.com/73725297/158000555-d0185aaf-767a-4efb-bdd9-99e17c93ff02.PNG)

Reply:
Yea, apologies. Thought I replied the other day. 
Yea, could you actually pull that out of the "contributing guidelines" and create a section called something like "Project Structure and Naming Conventions" 

Developers typically are horrible at keeping documentation up to date, but are the first to complain about it when the documentation is out of date. For this reason, we should try to keep our documentation as maintenanec free as possible. 

So, rather then providing alot of specific detail about what is in _every directory_ let's just give a high level overview (with an example) of what _should be _ at each echelon of organization 

Example:
Root level should contain any project level configuration and the src directory
inside src directory, we should include directories which act as 'modules' for example, the "components module" and the "core module" eventually we may have a "base" module, so-on. 
...etc

Reply:
Haha it's all good. Yeah I agree that it'd be best to remove all the small details about the project structure so that the documentation becomes easier to maintain. I'll rewrite it and post it here so that you can check.

Reply:
Awesome. 
That looks great.